sqlmesh/core/model/kind.py

```diff
@@ -288,11 +328,12 @@ def metadata_hash_values(self) -> t.List[t.Optional[str]]:
             *super().metadata_hash_values,
             str(self.batch_size) if self.batch_size is not None else None,
             str(self.forward_only),
+            str(self.on_destructive_change),
```

Comment:
Shouldn't this be a part of the `metadata_hash_values` in `_Incremental`? This way we don't need to duplicate this